docs/user_guide/resource_quota_mgmt.md

```diff
@@ -150,3 +150,38 @@ Jobs continue to be submitted to namespaces, based on the `Placementrule` used
 in the configuration. YuniKorn will automatically run the job and all its pods in
 the corresponding queue. For example, if a job is submitted to namespace `development`,
 then you will see the job is running in `root.development` queue.
+
+## Option 3) Custom mapping of workspaces to queues using Kubernetes shim
```

Comment:
I do not think we should use `workspace` here, this is not a native concept in K8s, people would get confused about this. Can we replace this with `Hierarchy queues with dynamical leaves that mapped to namespaces`

Reply:
Can we add the `Goal` and `Configuration` section as well?
In order to keep the format of the doc consistent. I suggest to have a structure like the following:

- Goal: what can be achieved via this setup? From the user's perspective
- Configuration: the example configuration in the current doc
- Set up namespaces: explain how to set up namespace to incorporate this config, e.g adding annotations.
- Run workloads: how to submit jobs to the structure just defined

so the user can simply following step by step to set this up.

Reply:
This is not very applicable here, as it makes more sense to first define the namespace config and only after the placement rules (the order is inverted following this structure), but I've modified it as you explained.